CustomEntitySearchTest/Constants.cs

```diff
@@ -0,0 +1,212 @@
 10046, -1, -1, 4562, -1, -1, 5463, -1, -1, -1, -1, -1, -1, -1, 381, -1, 9086, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 1192, -1, -1, -1, -1, -1, 3306, -1, -1, -1, -1, 2902, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 6942, -1, -1, -1, 2189, -1, 1003, -1, -1, -1, -1, -1, 3571, -1, -1, -1, -1, -1, -1, -1, -1, 84, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 1200, -1, -1, -1, -1, -1, 8290, -1, -1, 5976, -1, -1, 8596, 5842, -1, -1, 4624, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 1656, -1, -1, -1, -1, -1, 5677, -1, -1, -1, -1, 5099, -1, -1, -1, -1, 1329, -1, -1, -1, -1, -1, -1, -1, -1, 6032, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 2551, -1, -1, -1, -1, -1, 5107, 5944, -1, -1, -1, -1, -1, -1, -1, -1, -1, 5209, -1, -1, -1, -1, 7695, -1, -1, 1714, -1, -1, -1, -1, -1, -1, -1, -1, -1, 6005, -1, -1, -1, 7529, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 7573, -1, -1, -1, -1, -1, -1, 6204, 3080, -1, -1, -1, -1, 6780, -1, -1, -1, -1, -1, -1, -1, -1, 709, 4052, -1, -1, -1, -1, 3401, -1, -1, -1, 6648, -1, -1, 5319, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 5163, -1, -1, -1, -1, -1, -1, -1, 9657, -1, -1, -1, -1, -1, 3163, -1, -1, 2541, -1, -1, 6564, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 1025, -1, -1, -1, -1, -1, -1, -1, -1, 7117, -1, -1, -1, -1, -1, -1, -1, -1, 4551, -1, -1, 1178, -1, -1, -1, -1, -1, 4052, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 672, 1114, -1, -1, -1, -1, 1930, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 4949, 4926, 6320, -1, -1, -1, -1, -1, 7815, -1, -1, -1, -1, -1, -1, 2971, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 6917, -1, -1, -1, -1, 12232, -1, -1, -1, -1, -1, -1, -1, 8522, -1, -1, -1, 2363, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 7856, -1, 3738, -1, -1, -1, 8833, 8892, -1, -1, -1, -1, 5036, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 8290, -1, 8990, 1505, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 2133, -1, 5344, -1, -1, -1, -1, -1, -1, 515, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 422, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 8603, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 8858, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 35, -1, 7419, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 5726, -1, -1, -1, 5691, -1, -1, -1, 3124, -1, -1, -1, -1, -1, -1, -1, -1, 4056, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 3099, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 11826, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 3174, -1, -1, -1, -1, -1, -1, -1, -1, 249";
```

Comment:
use descriptive variable names

even its just like {TestName}ExpectedInput and {TestName}ExpectedOutput